src/core/toga/style/pack.py

```diff
@@ -96,27 +97,41 @@ def apply(self, prop, value):
                 )
 
     def layout(self, node, viewport):
-        self._layout_node(node, viewport.width, viewport.height, viewport.dpi)
+        print(viewport)
+        print(viewport.__dict__)
+        # `baseline_dpi` is missing in travertino's `Viewport`. Temporarily hack around
+        # that with `getattr()`. In the near future (before merging this code),
+        # let's do `viewport.baseline_dpi` not `getattr(viewport, 'baseline_dpi', ...)`.
+        self._layout_node(node, viewport.width, viewport.height, viewport.dpi, getattr(viewport, 'baseline_dpi', None))
         node.layout.content_top = node.style.padding_top
         node.layout.content_bottom = node.style.padding_bottom
 
         node.layout.content_left = node.style.padding_left
         node.layout.content_right = node.style.padding_right
 
-    def _layout_node(self, node, alloc_width, alloc_height, dpi):
+    def _layout_node(self, node, alloc_width, alloc_height, dpi, baseline_dpi):
```

Comment:
I wonder if a better approach here would be to pass in curried scaler (or a scaling object) at this point, rather than multiple dpi args. That would make code a lot simpler, as we would just invoke `scale()` that would incorporate a specific DPI and baseline DPI value